contracts/contracts/Poll.sol

```diff
@@ -237,8 +247,8 @@ contract Poll is Params, Utilities, SnarkCommon, Ownable, EmptyBallotRoots, IPol
   }
 
   /// @inheritdoc IPoll
-  function numSignUpsAndMessages() public view returns (uint256 numSignups, uint256 numMsgs) {
-    numSignups = extContracts.maci.numSignUps();
-    numMsgs = numMessages;
+  function numSignUpsAndMessages() public view returns (uint256 _numSignups, uint256 _numMessages) {
```

Comment:
I'd keep these variable names without underscore because on frontend they will be accessed by the names and it can conflict with code styles and linter. Let's rename them to avoid it.